extensions/blocks/podcast-player/edit.js

```diff
@@ -18,6 +18,7 @@ import {
 	ToggleControl,
 	Spinner,
 } from '@wordpress/components';
+import { useInstanceId } from '@wordpress/compose';
```

Comment:
I am afraid you won't be able to use `useInstanceId` yet. It is not available on sites running WordPress 5.3 (it was only added to Gutenberg a few months ago).

Reply:
What do you tend to use instead? Lodash `unique()`?

Reply:
Seems like `withInstanceId` is the next thing to try? https://github.com/WordPress/gutenberg/blob/master/packages/compose/src/higher-order/with-instance-id/README.md

Reply:
`withInstanceId` should work 👍

Reply:
Switched to `withInstanceId` 👍

Reply:
@jeryj Could you push your changes so I can give it a try?